react/components/Slide.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react'
+import EventListener from 'react-event-listener'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import debounce from 'debounce'
+import styles from './styles.css'
+import { NoSSR } from 'vtex.render-runtime'
+
+class SlideComponent extends Component {
+  constructor(props) {
+    super(props)
+    this.imgRef = React.createRef()
+    if (typeof window !== 'undefined') {
```

Comment:
```suggestion
    if (!!window) {
```
Transform undefined (`falsy`) into `false` would not work here?